OWML.ModHelper/ModConfig.cs

```diff
@@ -52,7 +53,7 @@ public void SetSettingsValue(string key, object val)
             if (value is JObject obj)
             {
                 obj["value"] = "" + val;
-            } 
+            }
```

Comment:
Nice

Reply:
I think my Visual Studio just does this automatically on save. Kinda miss the more thorough linting that I usually have on JavaScript / Typescript, though :P